index.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"> 
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet"/>
+    <link rel="stylesheet" href="index.css">
+    <title>EDULEARN CONFERENCE</title>
+</head>
+
+<body>
+<header>
+  <div class="top-nav"> 
+    <a href="#" target="_blank"><i class="fa-brands fa-facebook-f"></i></a>
+    <a href="#" target="_blank"><i class="fa-brands fa-twitter"></i></a>
+    <a href="#">English</a>
+    <a href="#">My Page</a>
+    <a href="#">LogOut</a>
+    
+    </div>
+<div class="menubar">
+   <div class="navbar">
+    <nav>
+        <img src="images/Union.png" class="navbarimg" alt="menu">
+    </nav>
+    </div>
+    <nav class="navbar1">
+    <div>  <img src="images/logo.png"  alt="logo for convocation"> 
+    </div>
+    <div> <ul>
+          <a  href="about.html">About</a>
+          <a  href="#courses">Courses</a>
+          <a  href="#speakers">Speakers</a>
+          <a  href="#partners">Sponsers</a> 
+          <a  href="#">News</a>    
+        </ul></div>
```

Comment:
- [ ] One more link to add as pointed earlier `CC CAMPAIGN`. 👍 
- [optional] I think you suppose to wrap each link `a` with `li` element for semantic purpose. 

## Your Design 
![Untitled](https://user-images.githubusercontent.com/103386183/219867842-ea40c29c-f869-45af-b7f6-d8f2690f7077.jpg)

## Template Design
![Untitled](https://user-images.githubusercontent.com/103386183/219868101-d9e8e947-8716-42a0-a594-694f9db90554.jpg)